meta-luneos/recipes-core/packagegroups/packagegroup-luneos-extended.bb

```diff
@@ -23,13 +23,14 @@ RDEPENDS_${PN} = " \
   pulseaudio-server \
   \
   alsa-conf \
+  ofono \
   tar \
   udev-extraconf \
+  voicecall \
   webos-connman-adapter \
+  webos-telephonyd \
   wireless-tools \
   \
-  ${@bb.utils.contains('MACHINE_FEATURES', 'phone', 'packagegroup-webos-telephony', '', d)} \
-  ${@bb.utils.contains('MACHINE_FEATURES', 'cellular', 'packagegroup-webos-telephony', '', d)} \
```

Comment:
Isn't it easier to simply add packagegroup-webos-telephony instead of each separate recipe?